tests/migrations/test_commands.py

```diff
@@ -1551,6 +1551,14 @@ def test_prune_respect_app_label(self):
             recorder.record_unapplied("migrations2", "0002_second")
             recorder.record_unapplied("migrations2", "0001_squashed_0002")
 
+    @override_settings(
+        INSTALLED_APPS=[
+            "migrations.migrations_test_apps.with_generic_model",
+        ]
+    )
+    def test_migrate_model_inherit_generic(self):
+        call_command("migrate", verbosity=0)
```

Comment:
When I checked adding makemigrations before this, it produced a RenameModel operation. Could you make the names agree?

Reply:
Oh my bad, I've just updated the migration file